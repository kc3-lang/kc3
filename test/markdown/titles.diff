--- titles.out.expected	Wed Sep  4 18:56:57 2024
+++ titles.out	Thu Sep  5 01:54:45 2024
@@ -1,4 +1,2 @@
-quote Markdown.titles("Text.\n")
-[]
-quote Markdown.titles("# Title\nText.\n\n## Title 2\n")
-["h1-1", "h2-1.1"]
+Markdown.titles("Text.\n")
+Segmentation fault (core dumped) 
--- titles.ret.expected	Wed Sep  4 18:57:08 2024
+++ titles.ret	Thu Sep  5 01:54:45 2024
@@ -1 +1 @@
-0
+139
