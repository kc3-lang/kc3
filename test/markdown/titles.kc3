quote Markdown.titles("Text.\n")
Markdown.titles("Text.\n")
quote Markdown.titles("# Title\nText.\n\n## Title 2\n")
Markdown.titles("# Title\nText.\n\n## Title 2\n")
