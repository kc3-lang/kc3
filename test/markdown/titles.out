Markdown.titles("Text.\n")
Segmentation fault (core dumped) 
